config/development.json

```diff
@@ -3,6 +3,13 @@
     "max_static_paths": 0,
     "revalidate": 360
   },
+  "dev_dot": {
+    "max_static_paths": 0,
+    "revalidate": 360
+  },
+  "learn": {
+    "max_static_paths": 0
+  },
```

Comment:
I don't feel super strongly about maintaining this split, but for now the granularity seemed nice as we iterate on each section.